lib/MusicBrainz/Server/Data/Artist.pm

```diff
@@ -119,6 +119,46 @@ sub find_by_area {
     );
 }
 
+sub find_by_instrument {
+    my ($self, $instrument_id, $limit, $offset) = @_;
+
+    # NOTE: if more tables than l_artist_recording are added here, check admin/BuildSitemaps.pl
+    my $query = "SELECT " . $self->_columns . ",
+                     array_agg(json_build_object('typeName', link_type.name, 'credit', lac.credited_as)) AS instrument_credits_and_rel_types
+                 FROM " . $self->_table . "
```

Comment:
Just a small thing, but to decrease the amount of indentation here we could start the `"SELECT` on the next line at 8 spaces. Might help with the long lines a bit.